spec/index.md

```diff
@@ -633,8 +641,64 @@ Calls must stay in `replied` or `rejected` long enough for polling users to catc
+
+The synchronous call endpoint is useful for users as it removes the networking overhead of polling the IC to determine the status of their call.
+
+The replica will maintain the HTTPS connection for the request and will respond once the state transitions to a terminal state. If an implementation specific timeout for the request is reached while the replica waits for the terminal state, then the replica will reply before the call completes, meaning a certificate for the call state in `unknown`, `received`, or `processing` is returned.
+
+The user should take the following actions for the returned certificate states:
+
+-    `processing`: The call is being executed by the IC. The user should poll the IC using [`read_state`](#http-read-state) requests to determine the result of the call.
+
+-    `received`: The call has been received by the IC. The user should poll the IC using [`read_state`](#http-read-state) requests to determine the result of the call.
+
+-    `unknown`: The user can conclude that the message was not received by the IC at the attached timestamp. The user can retry the call, or poll the IC using [`read_state`](#http-read-state) requests.
+
 ### Request: Call {#http-call}
 
+In order to call a canister, the user makes a POST request to `/api/v3/canister/<effective_canister_id>/call`. The request body consists of an authentication envelope with a `content` map with the following fields:
+
+-   `request_type` (`text`): Always `call`
+
+-   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
+
+-   `canister_id` (`blob`): The principal of the canister to call.
+
+-   `method_name` (`text`): Name of the canister method to call.
+
+-   `arg` (`blob`): Argument to pass to the canister method.
+
+The HTTP response to this request can have the following responses:
+
+-   200 HTTP status with a non-empty body. This status is returned if the replica received the message.
+    
+    -   A certificate for the state of the update call is produced, and returned as a CBOR (see [CBOR](#cbor)) map with the fields below. The user should use the certificate to determine the state of the call:
+
+        -   `status` (`text`): `"certified_state"`
+
+        -   `reply` (`blob`):  A certificate (see [Certification](#certification)) with subtrees at `/request_status/<request_id>` and `/time`. See [Request status](#state-tree-request-status) for more details on the request status.
+
+    -   If a non-replicated pre-processing error occurred due to the [canister inspect message](#system-api-inspect-message), then a body with information about the IC specific error encountered is returned. The body is a CBOR map with the following fields:
+
+        -   `status` (`text`): `"canister_inspect_message_rejected"`
```

Comment:
this is too specific, there can be multiple reasons why a pre-processing error occurs, e.g., the canister is frozen

Reply:
Changed it to "non_replicated_rejection"